.github/workflows/build-test-release.yml

```diff
@@ -150,36 +150,70 @@ jobs:
       fail-fast: false
       matrix:
         splunk-version: ["8.1","8.2"]
+    env:
+      KUBECONFIG: /etc/rancher/k3s/k3s.yaml
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Setup kubernetes cluster
+        id: kubernetes
+        run: |
+          ./k3s.sh
+          kubectl get no
+      - name: Setup Splunk-Operator at cluster-scope
+        run: |
+          wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
+          kubectl apply -f splunk-operator.yaml
+          sleep 30
```

Comment:
Is there a method to check that operator the previous operation was successful without sleeping 30 seconds? It can introduce flake failing pipeline.

Reply:
Yes, because after requesting the creation of a kubernetes resource it won't get scheduled on node. It can take some fraction of seconds to schedule then it will be able to access the logs of the pod. So, I have added the check for the same.